Doc/library/calendar.rst

```diff
@@ -586,10 +586,16 @@ The following options are accepted:
    or as an HTML document.
 
 
+.. option:: --firstweekday WEEKDAY, -f WEEKDAY
```

Comment:
Let's hyphenate this for readability and accessibility (for example, it helps people using screen readers):
```suggestion
.. option:: --first-weekday WEEKDAY, -f WEEKDAY
```

The GNU reference manual also recommends it:

> GNU adds _long options_ to these conventions. Long options consist of `--` followed by a name made of alphanumeric characters and dashes. Option names are typically one to three words long, with hyphens to separate words. Users can abbreviate the option names as long as the abbreviations are unique.


https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html

Reply:
I applied your changes.